key-manager/src/main/scala/org/bitcoins/keymanager/config/KeyManagerAppConfig.scala

```diff
@@ -30,12 +29,40 @@ case class KeyManagerAppConfig(
 
   lazy val networkParameters: NetworkParameters = chain.network
 
+  lazy val walletNameOpt: Option[String] = {
+    config.getStringOrNone(s"bitcoin-s.wallet.walletName")
+  }
+
   /** The path to our encrypted mnemonic seed */
   lazy val seedPath: Path = {
-    baseDatadir.resolve(WalletStorage.ENCRYPTED_SEED_FILE_NAME)
+    val prefix = walletNameOpt match {
+      case Some(walletName) =>
+        s"$walletName-"
+      case None => ""
+    }
+    baseDatadir
+      .resolve(WalletStorage.SEED_FOLDER_NAME)
+      .resolve(s"$prefix${WalletStorage.ENCRYPTED_SEED_FILE_NAME}")
   }
 
-  override def start(): Future[Unit] = FutureUtil.unit
+  override def start(): Future[Unit] = {
+    walletNameOpt match {
+      case Some(_) => FutureUtil.unit
```

Comment:
Hmm... so why don't we need to create it -- or at least see if it exists -- in this case too?

Reply:
No, that is handled by the wallet currently, we should probably move that to here in a future PR

Reply:
#2363